TypeScript.py

```diff
@@ -792,6 +795,29 @@ def on_close(self, view):
                     cli.service.close(view.file_name())
         logger.view_debug(view, "exit on_close")
 
+    def handlePulse(self):
+        view = active_view()
+        info = self.getInfo(view)
+        if info:
+            self.showTopLineStatus(info, view)
+        sublime.set_timeout(self.handlePulse, 500)
```

Comment:
Why it needs to re-check every 500ms? My understanding is icon changes can only happen if the content has been modified. Also if the use is idling, we should be able to detect that and end the loop?